src/cromshell/validate/command.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from pathlib import Path
+
+import click
+
+import cromshell.utilities.miniwdl_utils as miniwdl
+import cromshell.utilities.womtool_utils as womtool
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ValidationFailedError(Exception):
+    pass
+
+
+class MissingArgumentError(Exception):
+    pass
+
+
+@click.command(name="validate")
+@click.argument("wdl", type=click.Path(exists=True), required=True)
+@click.argument("wdl_json", type=click.Path(exists=True), required=False)
+@click.option(
```

Comment:
In submit this is `"--dependencies-zip`. We should probably make this match.  (Or maybe we should change it in submit?)

Reply:
Miniwdl allows users to provide more than one folder to look for dependencies. 
I can make it match the `submit`command so that users can only provide one folder (or zip) to send to miniwdl.

Reply:
done, so now you can provide a zip or a path to a directory and it will provide it to miniwdl. I noticed that it doesn't handle nested workflows well (when the main workflow is in a child directory separate from its imported workflows). However it should be possible to reuse the same logic in the nested wdl PR to resolve it.